packages/back-end/src/integrations/BigQuery.ts

```diff
@@ -47,7 +48,16 @@ export default class BigQuery extends SqlIntegration {
       useLegacySql: false,
     });
     const [rows] = await job.getQueryResults();
-    return rows;
+    const statistics = {
+      executionDurationMs: job.metadata.statistics.finalExecutionDurationMs,
+      totalSlotMs: job.metadata.statistics.totalSlotMs,
+      bytesProcessed: job.metadata.statistics.totalBytesProcessed,
+      bytesBilled: job.metadata.statistics.query.totalBytesBilled,
+      warehouseCachedResult: job.metadata.statistics.query.cacheHit,
+      partitionsUsed:
+        job.metadata.statistics.query.totalPartitionsProcessed > 0 ?? undefined,
```

Comment:
You have the type of this specified as a number, but are setting it to a boolean here